book/src/installers/customizing.md

```diff
@@ -0,0 +1,109 @@
+# Customizing
+
+> since 0.12.0
+
+axo's installers are designed to require no end user input- however, in many
+cases, end users like the ability to tailor their installation experience. For
+these users, our installers allow several inputs which give the user more
+control over their installation experience via configuration. Most of these are
+implemented using environment variables, however several are also available as
+arguments to the installer scripts themselves.
```

Comment:
Maybe add a comment that we recommend devs add a version of this info to their own docs if they expect their end users to want it?

Reply:
yeah this is a good call!